about.html

```diff
@@ -11,58 +11,95 @@
 
 </head>
 
+<header>
+    <nav>
+        <div class="navigation">
+       <img class= "logo" src="img/lambda-black.png" alt="">
+        <a href="index.html">Home</a>
+        <a href="about.html">About</a>
+        <a href="products.html">Products</a>
+        <a href="">Blog</a>
+        <a href="contact.html">Contact</a>
+    </div>
+    </nav>
+</header>
+
 <body>
     <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+        <img src="img/jumbo-about.png" alt="">
+<div class="about">
+    <h4>About</h4>
+      <p>    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+        </p>
+      </div>
 
 
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
+    <div class="strategy-collab">
+        <div class="square">
 
+        <img src="img/about-plan.png" alt="strategy">
 
+    <h4>
+    Strategy</h4>
+    
+    <p>
+    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+  </p>
+   <button>
+  Learn More</button> 
+</div>
 
+<div class="square">
     <img src="img/about-working.png" alt="strategy">
     
-    How We Work
+    <h4>
+    How We Work</h4>
     
+    <p>
     Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+  </p>
+    <button>
+    Learn More</button>
+  </div>
 
-    Learn More
-
-
-
+<div class="square">
     <img src="img/about-office.png" alt="strategy">
     
-    Places We Work
+    <h4>
+    Places We Work</h4>
 
+    <p>
     Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+  </p>
+    <button>
+    Learn More</button>
+  </div>
 
-    Learn More
-
+  <div class="square">
     <img src="img/about-meeting.png" alt="strategy">
     
-    Collaboration
+    <h4>Collaboration</h4>
     
+    <p>
     Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
+  </p>
+  <button>
+    Learn More</button>
+  </div>
+    </div>
+    <div class="work">
```

Comment:
```suggestion
    <div class="work">
```
For your sake, and the sake of future developers, it's usually good to be a little more semantic with you class names. :)